_layouts/contact_us.html

```diff
@@ -15,11 +15,17 @@
   
   {{ page.intro_content | markdownify }} 
   
+  {% if site.contact_phone_number_enabled and site.contact_phone_number and 
+     site.contact_phone_number_location == 'top' %} 
```

Comment:
I wonder if it'd be easier to manage the conditional output in the partial itself, and then the only condition we need here is for the location.

For example, in `contact_phone.html`:

```html
{% if site.contact_phone_number_enabled and site.contact_phone_number %}
  <h3 id="call-us">...
```

Reply:
It would certainly keep the conditional in once place. But it makes more sense – at least to me – to keep the logic on whether to show something in the layout, and limit the html file to purely just defining the html. Maybe I've been doing MVC for too long. :-)